.github/workflows/build.yml

```diff
@@ -422,6 +423,7 @@ jobs:
           chmod uog+x build/moonbeam-tracing
           cd moonbeam-types-bundle
           npm install
+          npm run build
```

Comment:
Why do you need to build then ? The tests are not using the built files.

Reply:
We just need to run it once to make sure we're not merging something that can't build